### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Noobware1**
Removed the previous code for iterable and list wrapper (sorry but there was too much to manually remove it all). 
Regenerated iterable and list wrapper with full implementation each.
Added Random wrapper.
Added MathTypes class.
Fixed StringSink.writeln writing null when called with 0 parameters.
Exported Iterator wrapper.

Feel free to point out the wrongs if there are any :)

### CODE REVIEW

**Comment by ethanblake4** [comment]
Oh cool, thanks for this!!

Only thing I'm noticing is that you removed $List.view(), please add it back, it serves a different purpose from the normal $List wrapper.

File: lib/src/eval/shared/stdlib/core/iterable.dart [hunk 1]
```diff
@@ -1,316 +1,1040 @@
...
-  Set<E> toSet() => $value.toSet();
+  E firstWhere(bool Function(E element) test, {E Function()? orElse}) =>
+      $value.firstWhere(test, orElse: orElse);
+  static const __$firstWhere = $Function(_$firstWhere);
+  static $Value? _$firstWhere(
+      Runtime runtime, $Value? target, List<$Value?> args) {
+    final $this = target?.$value as Iterable;
+    final test = args[0] as EvalCallable;
+    final orElse = args[1] as EvalCallable?;
+    print(orElse?.call(runtime, null, []));
```
Comment (ethanblake4) [inline lib/src/eval/shared/stdlib/core/iterable.dart:932]: Leftover print statement here

**Comment by Noobware1** [comment]
Done.

File: lib/src/eval/shared/stdlib/core/iterable.dart (hunk 1 above)
Comment (Noobware1) [reply lib/src/eval/shared/stdlib/core/iterable.dart]: Removed it.

**Pull request merged**